### ISSUE TITLE
can't figure out how to alternate between X and O

### ISSUE THREAD

**Issue by jillrizley** [ga-wdi-boston/game-project#791]
I've tried a handful of codes online that I am trying to use as a base to alternate, but nothing seems to be working. I was thinking of doing it for odd and even clicking, but can't figure out the code to make that happen, if I do that I think I need to loop it around 9 times, for each square. I also thought about defining the players, but I'm having a hard time working them into anything.

So far, I can get an x in any square with this code:
$('#game-board').on('click', function (event) {
  $(event.target).text('X', event.target)
})

But that's about it

**Comment by Jcornmanhomonoff** [comment]
It sounds like an if statement might be valuable here....

**Comment by BenGitsCode** [comment]
Have you tried pseudo coding it out? In plain English, what are you trying to accomplish right now? Something that switches players between turns?

**Comment by jillrizley** [comment]
yes!

// make an x appear in a box when clicked
```js
$('#game-board').on('click', function (event) {
  $(event.target).text('X', event.target)
})
```

// now to change every other x to an o
// if odd click, put X
// otherwise put an O
// loop it, 9 time? for all of the squares?

but I can't figure out how to actually write it out...this is kind of my on-going javascript issue with choosing what to put where and making names for things

**Comment by BenGitsCode** [comment]
Awesome! The key is just going slow and pseudocoding whenever in doubt. Don't worry about naming them, but what are some variables we need to use for that pseudocode? For instance, with `if odd click, put x` assumes some things. Do we have something that counts clicks? Do we have a variable that we can set to X or O?

I think that one line is a great starting place, but what do you need to start working towards accomplishing that and the next two lines?

**Comment by jillrizley** [comment]
so I've made some progress! now I am just trying to figure out how to get it to return an X or O!

let clickCounter = 0

$('#game-board').on('click', function (event) {
  $(event.target).text('X', event.target)
  clickCounter++
  clickCounter
  console.log(clickCounter)
  if (clickCounter % 2 === 0) {
    console.log('is even')
  } else {
    console.log('is odd')
  }
})

So far it is console logging both the number of clicks it has been through as well as if it is odd or even, but now I need to display that info on the board and am stuck there!

**Comment by BenGitsCode** [comment]
Awesome!

So you have the right behavior for your if statement. Let's break down that last statement into some smaller problems. You need that function to return X or O, based on turns taken, and render the respective tokens on the board. Cool, start with that very last part. How could you get an X / O to render on your gameboard? What HTML elements make up each cell of your gameboard, and how could get something to show up there. 

Focus on just getting something on the game board at all, then you can come back to getting the right thing to render.

**Comment by jillrizley** [comment]
by some magic, I got alternating x's and o's on my page....but now I have a bunch more issues....should I open new issues for them? or can they be handled here as well? They do relate to x and o placement

// only click in the table-right now, anywhere I click horizontally within the boxes height it will hide my boxes and put an x instead
// only click one square once-right now, I can click in a square infinitely and it will continue alternating
// only click until the game has been won-similar to the last problem, it will click forever! no game end, which I am sure I should be able to solve when I can declare a winner and end the game but I am unsure how to do so
// end the damn game

let clickCounter = 0

$('#game-board').on('click', function (event) {
  $(event.target).text('')
  clickCounter++
  clickCounter
  console.log(clickCounter)
  if (clickCounter % 2 === 0) {
    $(event.target).text('O')
  } else {
    $(event.target).text('X')
  }
})

**Comment by payne-chris-r** [comment]
Also, protip:
Try using "```js" in front of your code blocks. 
It changes:
const jill = 'isthatjavascript?!'
to
```js
const jill = 'ahhhyesitisjavascript'
```

**Comment by payne-chris-r** [comment]
To answer your first question, yes it should probably be a new issue.
Something along the lines of:
// only click in the table-right now, anywhere I click horizontally within the boxes height it will hide my boxes and put an x instead
Try to be methodical, think to yourself "What's the most _immediate_ problem at hand!?"
